04week/algorithms.js

```diff
@@ -9,22 +9,83 @@ function getRandomInt(min, max) {
 }
 
 let arr = [];
+const len = arr.length;
 
-for (let i = 0; i < 1000; i++) {
+for (let i = 0; i < 10; i++) {
   arr.push(getRandomInt(0, 1000));
+};
+
+function bubbleSort(arr){
+  let len = arr.length;
+  for (let i = len-1; i>=0; i--){
+    for(let j = 1; j<=i; j++){
+      if(arr[j-1]>arr[j]){
+        var temp = arr[j-1];
+        arr[j-1] = arr[j];
+        arr[j] = temp;
+      }
```

Comment:
Renata, I see where your going here, however, I would like for you to avoid using two loops and use a recursive function instead.